osu.Game/Database/LegacyScoreExporter.cs

```diff
@@ -20,6 +20,14 @@ public LegacyScoreExporter(Storage storage)
         {
         }
 
+        protected override string GetFilename(ScoreInfo score)
+        {
+            string scoreString = score.GetDisplayString();
+            string filename = $"{scoreString} ({score.Date.LocalDateTime:yyyy-MM-dd})";
```

Comment:
One final change I'd propose is adding a time to this as well, since right now it would not be easy to tell which score is which when viewing files (and not even ascertain the order of achievement over a day).

```suggestion
            string filename = $"{scoreString} ({score.Date.LocalDateTime:yyyy-MM-dd_hh-mm})";
```